docs/getting-started.md

```diff
@@ -0,0 +1,70 @@
+# Getting started
+
+## Prerequisites
+
+You'll need the following tools installed in order to run the project locally:
+
+-   [Composer](https://getcomposer.org/download/) v1.3.0+
+-   MySQL v5.7+ or PostgreSQL 9.5+
+
+Once you have the prerequisites installed on your machine, and have the project checked out locally, run these commands in your terminal of choice:
+
+## Install dependencies
+
+From the root of the project run:
+
+```shell script
+composer install
+```
+
+## Create a Database
+
+Next up, you need to create a database for the project. Both MySQL 5.5+ and PostgreSQL 9.5+ are supported.
+
+If you’re given a choice, we recommend the following database settings in most cases:
+
+-   MySQL
+
+    -   Default Character Set: utf8
+    -   Default Collation: utf8_unicode_ci
+
+-   PostgreSQL
+
+    -   Character Set: UTF8
+
+## Create a local `.env` file
+
+This is used to set local configuration values. There is a `.env.sample` file at the root of the project which documents available options. You should make a copy of this as `.env`.
+
+You need to set all values to point to the local database instance you set up.
+
+## Set up the Web Server
+Create a new web server to host the project. Its document root (or “webroot”) should point to the `web/` directory.
+
+If you’re not using MAMP or another localhosting tool, you will probably need to update your hosts file, so your computer knows to route requests to your chosen host name to the local computer.
+
+-   macOS/Linux/Unix: `/etc/hosts`
+-   Windows: `\Windows\System32\drivers\etc\hosts`
+
+You can test whether you set everything up correctly by pointing your web browser to `http://<Hostname>/index.php?p=admin/install` (substituting <Hostname> with your web server’s host name). If Craft’s Setup Wizard is shown, the host name is correctly resolving to your Craft installation.
+
+## Run the Setup Wizard
+Finally, it’s time to run Craft’s Setup Wizard. You can either run that from your [terminal](###Terminal Setup) or your [web browser](###Web Browser Setup).
+
+###Terminal Setup
```

Comment:
This and the web browser set up heading need a space before the text to format correctly, worth double checking this by previewing the rendered file on github.